docs/src/modeler_guide/enumerated_types.md

```diff
@@ -0,0 +1,100 @@
+
+# Enumerated Types
+
+Some of the fields of `Component` structs are specified with `Enum` types. This is useful
+when the field should specify an option from a pre-defined list. The following are the
+enumerated types contained in `PowerSystems`
+
+## `ThermalFuels`
+
+Each `ThermalGen` generator struct contains a field for `fuel::ThermalFuels` where `ThermalFuels`
+are intended to reflect the options denoted by the
+[Aggregated Fuel Codes](https://www.eia.gov/survey/form/eia_923/instructions.pdf) from the
+EIA Annual Energy Review. Specifically, `ThermalFuels` is an enumerated type with the
+following options:
+
+| EnumName | EnumValue | EIA Fuel Code | Description |
```

Comment:
Not sure if it makes sense to have the `EnumValue` column or not. @jd-lara  and @daniel-thom let me know what you think.

Reply:
Probably not. No one should ever need or use them.